wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -550,6 +551,13 @@ def _create_run(self) -> Dict[str, Any]:
             )
         return {}
 
+    def _set_sweep_state(self, state: str) -> None:
```

Comment:
This seems like it should also be used elsewhere? Will there be a follow up with other uses for this?

Reply:
This only happens when the sweep scheduler ends when it hits a run_cap (previously managed by gorilla, but now handled by the scheduler). We then don't mark the sweep as finished in the backend. I think this is the only case where this happens...

Reply:
JK I guess we also have to call this when the scheduler starts...